### PR TITLE
Fix/node build

### PR DESCRIPTION
**Pull request by lemoustachiste**
Moving to a rollup generated build to prevent CJS/Module conflicts.
Adding a proper node build test by running against a server.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,25 +8,25 @@
   },
   "main": "dist/verifier.js",
   "module": "dist/verifier-es.js",
+  "node": "dist/verifier-node.js",
   "typings": "dist/index.d.ts",
   "scripts": {
     "dts:bundle": "dts-bundle-generator -o ./dist/index.d.ts --project tsconfig.json --no-banner src/index.ts",
     "test": "jest --setupFilesAfterEnv=./test/setup.js",
-    "test:build": "npm run test:build:esm && npm run test:build:iife",
+    "test:build": "npm run test:manual:node:server& npm run test:build:esm && npm run test:build:iife && kill -9 $(lsof -t -i:4000)",
```
Comment (raiseandfall) [inline package.json:16]: `npm run test:manual:node:server && npm run test`

That should run but just for consistency.

**Review by raiseandfall — CHANGES_REQUESTED** [review]
Just that ampersand tiny fix. Otherwise LGTM

File: package.json (hunk 1 above)
Comment (lemoustachiste) [reply package.json:16]: no it's done on purpose to put the server process in the background and let the tests run:
https://www.linux.com/training-tutorials/and-ampersand-and-linux/

**Pull request merged**